### PR TITLE
Doc fixes

### PR DESCRIPTION
**Pull request by Dav0815**
**Description:**
Fixes to documentation standards. 

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#<home-assistant PR number goes here>

## Checklist:

- [x] Branch: `next` is for changes and new documentation that will go public with the next [home-assistant](https://github.com/home-assistant/home-assistant) release. Fixes, changes and adjustments for the current release should be created against `current`.
- [x] The documentation follow the [standards][standards].

[standards]: https://home-assistant.io/developers/documentation/standards/

### CODE REVIEW

File: source/_components/sensor.swiss_public_transport.markdown [hunk 1]
```diff
@@ -30,10 +30,21 @@ sensor:
...
+to:
+  description: The ID of the station of the end station.
+  required: true
+  type: string
+  default: 
+name:
+  description: The name of the sensor. 
+  required: false
+  type: string
+  default: Defaults to 'Next Departure'.
```
Comment (frenck) [inline source/_components/sensor.swiss_public_transport.markdown]: Just put `Next Departure` in here.

File: source/_components/sensor.ted5000.markdown [hunk 2]
```diff
@@ -24,11 +24,23 @@ sensor:
...
+port:
+  description: The port of your ted gateway.
+  required: false
+  type: integer
+  default: 80
+name:
+  description: Name of the ted gateway.
+  required: false
+  type: string
+  default: Default is ted.
```
Comment (frenck) [inline source/_components/sensor.ted5000.markdown]: Just put `ted` in here.

File: source/_components/sensor.worldclock.markdown [hunk 3]
```diff
@@ -26,10 +26,17 @@ sensor:
...
+{% configuration %}
+time_zone:
+  description: The resource or endpoint that contains the value.
+  required: true
+  type: string
+name:
+  description: The name of the sensor, eg. the city.
+  required: false
+  type: string
+  default: Defaults to 'Worldclock Sensor'.
```
Comment (frenck) [inline source/_components/sensor.worldclock.markdown]: Just put `Worldclock Sensor` in here.

File: source/_components/sensor.worxlandroid.markdown [hunk 4]
```diff
@@ -25,8 +25,18 @@ sensor:
...
+  type: string
+pin:
+  description: The pin code for the mower.
+  required: true
+  type: integer
+allow_unreachable:
+  description: This will allow the mower to be outside of wifi range without raising an error.
+  required: false
+  type: boolean
+  default: True
```
Comment (frenck) [inline source/_components/sensor.worxlandroid.markdown]: `True` -> `true`

**Review by frenck — CHANGES_REQUESTED** [review]
Thanks, @Dav0815 👍 
I've reviewed it, and left you a couple of small comments. Could you please take a look? 👍

File: source/_components/sensor.swiss_public_transport.markdown (hunk 1 above)
Comment (Dav0815) [reply source/_components/sensor.swiss_public_transport.markdown]: Done

**Review by frenck — APPROVED** [review]

**Comment by frenck** [comment]
Thanks for updating the PR @Dav0815! 👍 

Happy Hacktoberfest 🎉 

../Frenck

**Pull request merged**